README.md

```diff
@@ -1,19 +1,92 @@
 # Official VOLTTRON docker image
 
 # Introduction
-This image provides a reproducable way to install VOLTTRON within a docker container.  It features gosu which allows the storage of VOLTTRON_HOME to be persistable on the hosts hard drive.  The
-base docker images are available on docker hub at https://hub.docker.com/r/volttron/volttron/.
+This image provides a reproducable way to install VOLTTRON within a docker container.  It features gosu which allows the storage of VOLTTRON_HOME to be persistable on the hosts hard drive.
```

Comment:
It seems I had an error in here for correctness.  `gosu` allows the volttron user inside the container to have the same UID as the user running outside the container.  This allows changes inside the container to be persisted outside and changes made outside the container to be reflected inside. 

Not really sure how to incorporate that but this is something I wanted originally to specify.